app/Http/Resources/UserResource.php

```diff
@@ -23,6 +23,12 @@ public function toArray(Request $request): array
             'proyectos' => ProyectoResource::collection($this->proyectos),
             'competencias' => CompetenciaResource::collection($this->competencias),
             'ciclos' => CicloResource::collection($this->ciclos),
+            'attachments' => $this->avatar
+            ? [
+                'src' => asset('storage/' . $this->avatar),
```

Comment:
No pasaría nada en principio, pero por si acaso dejarlo de la siguiente manera:
`'src' => asset('/storage/' . $this->avatar),`